### DISCUSSION TITLE
Multiple Editors, drag and drop creates copies.

### DISCUSSION THREAD

**Discussion by LyDawei** [ueberdosis/tiptap#3097] [General Questions]
Hi, I have multiple editors on screen with a requirement to drag and drop between the editors in a ReactJS app.

If I drag from an editor and drop into another editor, while I do receive the appropriate node in the new editor, the old one in the other editor remains. I've been trying to figure this one out and thought I could maybe work through the handleDrop, by figuring out where the dropped element came from by attaching the source Editor to the DragEvent's dataTransfer, and then sifting through the old editor, and removing the content that was passed from the Slice, but inside the editor config's handleDrop event, I do not have access to my other editors.

**Comment by colinramsay** [comment]
https://github.com/ueberdosis/tiptap/pull/3429

I think this might be the feature you are looking for?
